challenges/prototypes.js

```diff
@@ -22,11 +22,12 @@ function  CuboidMaker(props) {
   this.length = props.length;
   this.width = props.width;
   this.height = props.height
-  this.volume = function () {
-    return (props.length * props.width * props.height);
-  };
+  
 }
 
+CuboidMaker.prototype.volume = function () {
+  return (props.length * props.width * props.height);
```

Comment:
Below, you are using `this.length`, but here you're using `props.length` is this a typo?  Which one is correct?  What does `props` refer to?